Assets/Prefabs/Player/PlayerVariation.prefab

```diff
@@ -259,8 +259,9 @@ MonoBehaviour:
   syncMode: 0
   syncInterval: 0.1
   movementDeadZone: 0.001
-  turningDeadZone: 0.001
+  turningDeadZone: 15
```

Comment:
This is the threshold change that temporarily disabled the turning animation. Should probably be replaced with something smarter in the future, will talk more about this idea in the future.